app/ui/qml/Paper.qml

```diff
@@ -76,7 +81,12 @@ Item {
 
 	KeyboardHandler {
 		id: keyboardHandler
-		onKeyPressed: control.keyPressed(event);
+		onKeyPressed: {
+			if(event["key"] === Qt.Key_Tab)
```

Comment:
Try to use `event.key`

Reply:
The 'event' variable is actually a QVariantMap (or object in JavaScript). I've used this as a temporary workaround because of the need of a custom key event. See the QmlKeyboardHandler class.

I'm planning to create a class for key events so that I can pass custom key events from C++ to QML.

Reply:
Oh, okay, I thouht that it is Qt class.